src/Stratis.Bitcoin/Controllers/RestApiClientBase.cs

```diff
@@ -107,9 +107,17 @@ public RestApiClientBase(ILoggerFactory loggerFactory, IHttpClientFactory httpCl
         public async Task<Response> SendGetRequestAsync<Response>(string apiMethodName, string arguments = null,
             CancellationToken cancellation = default(CancellationToken)) where Response : class
         {
-            HttpResponseMessage response = await this.SendGetRequestAsync(apiMethodName, arguments, cancellation).ConfigureAwait(false);
+            try
+            {
+                HttpResponseMessage response = await this.SendGetRequestAsync(apiMethodName, arguments, cancellation).ConfigureAwait(false);
 
-            return await this.ParseHttpResponseMessageAsync<Response>(response).ConfigureAwait(false);
+                return await this.ParseHttpResponseMessageAsync<Response>(response).ConfigureAwait(false);
+            }
+            catch (Exception e)
+            {
+                Console.WriteLine(e);
```

Comment:
hmmmm….. you have a logger

Reply:
Ah, yes, forgot to remove that. I used it during tests debugging.